spec/PointerPermissions.spec.js

```diff
@@ -171,6 +171,47 @@ describe('Pointer Permissions', () => {
     })
   });
 
+  it('should query on pointer permission enabled column', (done) => {
```

Comment:
@flovilmart Just wanted to double check this test case was failing before when you put it in. I'm assuming it was, just checking that we aren't fixing ghosts.

Reply:
I added this because that would have been probably failing depending on the implementation of this change: https://github.com/parse-community/parse-server/pull/4141/files#diff-26b0f888e8a20f22d9728a26272f6cddR945